src/__tests__/server-smoke.test.ts

```diff
@@ -0,0 +1,31 @@
+import { startServer, waitForEbb } from '../server';
+import type { Server } from 'http';
+import request from 'supertest';
```

Comment:
I feel like we probably don't need an extra dependency to make an HTTP request. Node's built-in `fetch` should be sufficient.